src/common/server.c

```diff
@@ -1568,7 +1568,6 @@ server_connect (server *serv, char *hostname, int port, int no_login)
 			port = 6697;
 #endif
```

Comment:
Throw a `g_debug ("Attempted to connect to invalid port, assuming default %u", port);` in there.

Reply:
Done.